### DISCUSSION TITLE
Borg Backup - Failed backup

### DISCUSSION THREAD

**Discussion by jjanszen01** [nextcloud/all-in-one#2069] [Questions] [answered]
### Steps to reproduce
1. Enable nightly backups
2. Wait
3. Failure

### Expected behavior 
Backups should happen nightly

### Actual behavior 
Backups fail nightly, initial backup works correctly. 

### Host OS 
Debian

#### Nextcloud AIO version 
4.4.1

#### Current channel 
latest

#### Other valuable info 
Backups are configured for - /mnt/backup/borg
Log (below) of failed backup show it trying to write to - /mnt/borgbackup/borg/ (This doesn't exist nor have I specified it anywhere)

Performing backup...
Starting the backup...
Failed to create/acquire the lock /mnt/borgbackup/borg/lock.exclusive (timeout).
Deleting the failed backup archive...
Failed to create/acquire the lock /mnt/borgbackup/borg/lock.exclusive (timeout).
Backup failed!

**Comment by szaimen** [comment, marked as answer]
> show it trying to write to - /mnt/borgbackup/borg/ (This doesn't exist nor have I specified it anywhere)

This error message is misleading. It is still writing to your configured directory on your host.

`sudo docker inspect nextcloud-aio-borgbackup` outputs:
```
...
            {
                "Type": "bind",
                "Source": "/mnt/backup",
                "Destination": "/mnt/borgbackup",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            },
...
```
So everything working fine there.

---

As for your issue, it seems like the borg lock was not removed at some point. Maybe you've shut down your server while a backup was running? Usually borg will release the lock automatically.

You should be able to fix this by installing borg on your host OS and then running `sudo borg break-lock /mnt/backup/borg`.
